src/get_started.rst

```diff
@@ -88,6 +88,21 @@ Apart from the compilation environment all other dependencies are handled the br
   # Install NetworKit via brew
   brew install python3
 
+Apple Silicon M1 hardware
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With the recently introduced :code:`macos/arm64` platform, it is possible to build and use both `arm64` and `x86_64` packages (via Rosetta2). While the later works without major modifications, arm-packages must be currently build from source using conda environments (e.g. miniforge).
```

Comment:
Is `macos/arm64` an official name? The official name for the architecture is `AArch64`, not `arm64`.

Reply:
Instead of mentioning Rosetta first, I'd restructure this to focus on native builds. You can add the sentence about Rosetta, too, but that'd be something along the lines of "Note that it is also possible to run `x86_64` packages of NetworKit via Rosetta2; however, this will likely incur some performance overhead.".

Reply:
> Is `macos/arm64` an official name? The official name for the architecture is `AArch64`, not `arm64`.

It is only a detail, but I also found this one tricky. Found no PEP or similar with specific naming conventions for platform tags on macOS. Linux-builds for ARM-chips are tagged with `aarch64`. Apple M1 packages however use `arm64` as platform tag (or `universal2`). Also the clang-compiler and Xcode-framework labels itself with `arm64-apple-darwin<VERSION>`. In the end I thought an easy way out would be to call it a "arm64 platform", but maybe thats too vague.

Reply:
> Instead of mentioning Rosetta first, I'd restructure this to focus on native builds. You can add the sentence about Rosetta, too, but that'd be something along the lines of "Note that it is also possible to run `x86_64` packages of NetworKit via Rosetta2; however, this will likely incur some performance overhead.".

Good point, changed accordingly.